14th-team5-iOS/Bibbi/Sources/Presentation/Notification/ViewControllers/NotificationViewController.swift

```diff
@@ -201,7 +204,9 @@ final class NotificationViewController: BBNavigationViewController<NotificationR
     override func setupUI() {
         super.setupUI()
         
-        contentView.addSubview(tableView)
+        contentView.addSubviews(
+            tableView
+        )
```

Comment:
```suggestion
        contentView.addSubview(tableView)
```
뷰 하나만 추가하면 이렇게 해도 될 것 같아요